Documentation/ceph-osd-mgmt.md

```diff
@@ -67,20 +67,21 @@ If you are using `useAllDevices: true`, no change to the CR is necessary.
 
 ### PVC-based cluster
 
-If you want to reduce the storage in your cluster, a healthy PVC-based OSD can be removed as described in this section.
+To reduce the storage in your cluster or remove a failed OSD on a PVC:
 
 1. Shrink the number of OSDs in the `storageClassDeviceSet` in the CephCluster CR.
    - `kubectl -n rook-ceph patch CephCluster rook-ceph --type=json -p '[{"op": "replace", "path": "/spec/storage/storageClassDeviceSets/0/count", "value":<desired number>}]'`
```

Comment:
It might be good to note that the `0` here is the index of the particular StorageClassDeviceSet if they have multiple.